### PR TITLE
Add missing messages in inflateMessage

### PR DESCRIPTION
**Pull request by gagbo**

### CODE REVIEW

**Comment by gagbo** [comment]
Ready for review. This crashes synchronization if the missing types aren't done (sorry for the whitespace changes though)

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp [hunk 1]
```diff
@@ -75,85 +75,102 @@ namespace ledger {
...
+                auto &content = boost::get<cosmos::MsgWithdrawDelegationReward>(msg.content);
+                content.delegatorAddress = row.get<std::string>(COL_DELADDR);
+                content.validatorAddress = row.get<std::string>(COL_VALADDR);
+            } break;
+            case api::CosmosLikeMsgType::MSGMULTISEND: {
+                msg.content = cosmos::MsgMultiSend();
+            } break;
+            case api::CosmosLikeMsgType::MSGCREATEVALIDATOR: {
+                msg.content = cosmos::MsgCreateValidator();
+
```
Comment (ghost) [inline ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:142]: Add // TODO ?

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp [hunk 2]
```diff
@@ -75,85 +75,102 @@ namespace ledger {
...
+            } break;
+            case api::CosmosLikeMsgType::MSGEDITVALIDATOR: {
+                msg.content = cosmos::MsgEditValidator();
+
```
Comment (ghost) [inline ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:146]: Add // TODO ?

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp [hunk 3]
```diff
@@ -75,85 +75,102 @@ namespace ledger {
...
+                soci::stringToCoins(row.get<std::string>(COL_AMOUNT), content.amount);
+            } break;
+            case api::CosmosLikeMsgType::MSGWITHDRAWDELEGATIONREWARD: {
+                msg.content = cosmos::MsgWithdrawDelegationReward();
+                auto &content = boost::get<cosmos::MsgWithdrawDelegationReward>(msg.content);
+                content.delegatorAddress = row.get<std::string>(COL_DELADDR);
+                content.validatorAddress = row.get<std::string>(COL_VALADDR);
+            } break;
+            case api::CosmosLikeMsgType::MSGMULTISEND: {
+                msg.content = cosmos::MsgMultiSend();
```
Comment (ghost) [inline ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:138]: Add // TODO ?

**Review by ghost — APPROVED** [review]
All good, just maybe flag missing implementations.

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp (hunk 1 above)
Comment (gagbo) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:142]: We'll see the empty message on the live side if necessary. I don't expect the history to need to handle these messages yet

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp (hunk 2 above)
Comment (gagbo) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:146]: We'll see the empty message on the live side if necessary. I don't expect the history to need to handle these messages yet

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp (hunk 3 above)
Comment (gagbo) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:138]: We'll see the empty message on the live side if necessary. I don't expect the history to need to handle these messages yet

Comment (gagbo) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:138]: But I think I actually changed that in another PR

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp (hunk 2 above)
Comment (ghost) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:146]: Ok :)

File: ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp (hunk 3 above)
Comment (gagbo) [reply ledger-core-cosmos/src/cosmos/database/CosmosLikeTransactionDatabaseHelper.cpp:138]: I did not, this is a new ticket

**Pull request merged**